src/components/safe-apps/SafeAppIconCard/index.tsx

```diff
@@ -46,21 +32,14 @@ const SafeAppIconCard = ({
   height?: number
   fallback?: string
 }): ReactElement => {
-  if (_isSafeSrc(src)) {
-    return <ImageFallback src={src} alt={alt} width={width} height={height} fallbackSrc={fallback} />
-  }
-
   return (
-    <iframe
-      title={alt}
-      srcDoc={getIframeContent(src, width, height, fallback)}
-      sandbox="allow-scripts"
-      referrerPolicy="strict-origin"
+    <ImageFallback
+      src={src}
+      alt={alt}
       width={width}
       height={height}
-      style={{ pointerEvents: 'none', border: 0 }}
-      tabIndex={-1}
-      loading="lazy"
+      fallbackSrc={fallback}
+      crossOrigin={_isSafeSrc(src) ? undefined : 'anonymous'}
```

Comment:
I would expect that we show the fallback image in case it is an external source. However, from my tests so far the behaviour is that the network request doesn't fail but the image is also not shown. Since the network request didn't fail we don't show the fallback image so it looks broken in the app.

Should it instead say: `src={_isSafeSrc(src) ? src : undefined}`?

<img width="832" alt="Screenshot 2024-08-05 at 15 33 27" src="https://github.com/user-attachments/assets/5b7088f7-b190-4972-8fea-4b05afc79467">

Reply:
Some sites support `crossorigin=anonymous`, some don't. I think ideally it should handle this as an error.

Reply:
@usame-algan I decided to remove both `crossorigin=anonymous` entirely, as well as `isSafeSrc` because this will not work for forks hosted on other domains. Let's assume config service icons are safe, and custom apps are added at user's own risk.